cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/metric/MetricType.java

```diff
@@ -34,8 +34,30 @@
   BROKER_PRODUCE_REQUEST_RATE((byte) 16),
   BROKER_CONSUMER_FETCH_REQUEST_RATE((byte) 17),
   BROKER_FOLLOWER_FETCH_REQUEST_RATE((byte) 18),
-  BROKER_REQUEST_HANDLER_AVG_IDLE_PERCENT((byte) 19);
-
+  BROKER_REQUEST_HANDLER_AVG_IDLE_PERCENT((byte) 19),
+  BROKER_REQUEST_QUEUE_SIZE((byte) 20),
+  BROKER_RESPONSE_QUEUE_SIZE((byte) 21),
+  BROKER_PRODUCE_REQUEST_QUEUE_TIME_MS_MAX((byte) 22),
+  BROKER_PRODUCE_REQUEST_QUEUE_TIME_MS_MEAN((byte) 23),
+  BROKER_CONSUMER_FETCH_REQUEST_QUEUE_TIME_MS_MAX((byte) 24),
+  BROKER_CONSUMER_FETCH_REQUEST_QUEUE_TIME_MS_MEAN((byte) 25),
+  BROKER_FOLLOWER_FETCH_REQUEST_QUEUE_TIME_MS_MAX((byte) 26),
+  BROKER_FOLLOWER_FETCH_REQUEST_QUEUE_TIME_MS_MEAN((byte) 27),
+  BROKER_PRODUCE_TOTAL_TIME_MS_MAX((byte) 28),
+  BROKER_PRODUCE_TOTAL_TIME_MS_MEAN((byte) 29),
+  BROKER_CONSUMER_FETCH_TOTAL_TIME_MS_MAX((byte) 30),
+  BROKER_CONSUMER_FETCH_TOTAL_TIME_MS_MEAN((byte) 31),
+  BROKER_FOLLOWER_FETCH_TOTAL_TIME_MS_MAX((byte) 32),
+  BROKER_FOLLOWER_FETCH_TOTAL_TIME_MS_MEAN((byte) 33),
+  BROKER_LOG_FLUSH_RATE((byte) 34),
+  BROKER_LOG_FLUSH_TIME_MS_MAX((byte) 35),
+  BROKER_LOG_FLUSH_TIME_MS_MEAN((byte) 36),
+  BROKER_PRODUCE_LOCAL_TIME_MS_MAX((byte) 37),
```

Comment:
Nit: Can we move the log flush rate/time to be after the local time so the request related times sits together.